src/views/PurchaseOrders.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <ion-page>
+    <ion-header :translucent="true">
+      <ion-toolbar>
+        <ion-title>{{ $t("Purchase Orders") }}</ion-title>
+      </ion-toolbar>
+    </ion-header>
+    <ion-content :fullscreen="true">
+      <div>
+        <ion-searchbar :placeholder="$t('Search')" />
```

Comment:
Implement search functionality, for now, search using the `orderId` and `externalOrderId` and confirm once with @dt2patel Sir for the searchable fields.

Reply:
Ideally if we are getting PO from SOLR anything indexed know SOLR should be searchable right?

Reply:
Yes Sir